ui/components/app/flask/experimental-area/index.scss

```diff
@@ -0,0 +1,44 @@
+.experimental-area {
+  color: $flask-purple;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+
+  .logo {
+    padding: 20px;
+    size: 10px;
```

Comment:
I think this is meant to be `font-size`; also, we try to stick to `rem` sizing instead of `px`; in this case, the value should be `0.625rem`